CANTroller2/src/devices.h

```diff
@@ -911,7 +999,7 @@ class Simulator {
         static constexpr bool initial_sim_starter = true;
         static constexpr bool initial_sim_ignition = true;
         static constexpr bool initial_sim_airflow = false;
-        static constexpr bool initial_sim_battery = true;
+        static constexpr bool initial_sim_battery = false;
```

Comment:
Why is this change in this PR? Let's try to track reasons for changes like this in the commit history or the git conversation or something

Reply:
Soren hacked the battery signal to double as our ignition check, so we decided not to simulate the battery. I put it here since i was already changing battery code and I would have had to special case some stuff to make the conversion valid with simulation.